internal/common/logger.go

```diff
@@ -76,6 +76,8 @@ func LogIssues(v1 *viper.Viper, source string) (*log.Logger, error) {
 	logger.SetReportCaller(false)                                   // disable include file name and line number
 	//mw := io.MultiWriter(os.Stdout, logFile)
 	//log.SetOutput(mw)
```

Comment:
Do you mind removing the commented out `SetOutput` statements in here? They make it look like there's a lot going on.